packages/client/src/rpc/modules/eth.ts

```diff
@@ -1119,4 +1204,75 @@ export class Eth {
 
     return bigIntToHex(gasPrice)
   }
+
+  async feeHistory(params: [string | number | bigint, string, Array<number>?]) {
```

Comment:
I don't think the `params` type is correct here. It should be `[bigint, string, [bigint]?]` per the spec